llvm/test/CodeGen/SystemZ/builtin-longjmp-01.ll

```diff
@@ -0,0 +1,24 @@
+;Test  longjmp load from jmp_buf.
+; Frame pointer from Slot 1.
+; Jump address from Slot 2.
+; Stack Pointer from Slot 4.
+; Literal Pool Pointer from Slot 5.
+
+; RUN: llc  < %s -mtriple=s390x-linux-gnu -O2 | FileCheck %s
+
+@buf = dso_local global [20 x ptr] zeroinitializer, align 8
+
+; Function Attrs: noreturn nounwind
+define dso_local void @foo() local_unnamed_addr #0 {
```

Comment:
We shouldn't need most of those annotations here.  We do not need the Function Attrs comment, nor the `#0` attribute reference (which doesn't even exist in this file!).  We also do not need the `dso_local` or `local_unnamed_addr` markers.